packages/moonstone/Button/tests/Button-specs.js

```diff
@@ -4,19 +4,6 @@ import Button from '../Button';
 
 describe('Button', () => {
 
-	test('should render with button text upper-cased', () => {
```

Comment:
We'll need to update all the UI and screenshot test cases that rely on the presence of the `Uppercase` HOC.